torch/lib/c10d/ProcessGroupMPI.cpp

```diff
@@ -588,6 +588,167 @@ std::shared_ptr<ProcessGroup::Work> ProcessGroupMPI::reduce_scatter(
   throw std::runtime_error("ProcessGroupMPI does not support reduce_scatter");
 }
 
+std::shared_ptr<ProcessGroup::Work> ProcessGroupMPI::alltoall_base(
+    std::vector<at::Tensor>& outputTensors,
+    std::vector<at::Tensor>& inputTensors,
+    std::vector<int>& outputSplitSizes,
+    std::vector<int>& inputSplitSizes,
+    const AllToAllOptions& opts) {
+  checkSingleTensor(outputTensors);
+  checkSingleTensor(inputTensors);
+
+  if(outputSplitSizes.size() == 0 && inputSplitSizes.size() == 0) {
```

Comment:
nit: I think we prefer to use "if (" and "for (" spacing instead of "if(" and "for("

Reply:
Ok, will fix this.

Reply:
We usually use `clang-format` for code formats. :)